src/Neo/SmartContract/Native/StdLib.cs

```diff
@@ -263,5 +264,44 @@ private static int StrLen([MaxLength(MaxInputLength)] string str)
 
             return count;
         }
+
+        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 13, RequiredCallFlags = CallFlags.None)]
```

Comment:
```suggestion
        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 13)]
```

Just shorter version.